great_expectations/experimental/column_descriptive_metrics/batch_inspector_agent_action.py

```diff
@@ -0,0 +1,44 @@
+from great_expectations.agent.actions import ActionResult, AgentAction
+from great_expectations.agent.models import (
+    CreatedResource,
+    RunBatchInspectorEvent,
+)
+from great_expectations.experimental.column_descriptive_metrics.batch_inspector import (
+    BatchInspector,
+)
+from great_expectations.experimental.column_descriptive_metrics.cloud_data_store import (
+    CloudDataStore,
+)
+from great_expectations.experimental.column_descriptive_metrics.metric_repository import (
+    MetricRepository,
+)
+
+
+class RunBatchInspectorAction(AgentAction[RunBatchInspectorEvent]):
```

Comment:
ColumnDescriptiveMetrics action

Reply:
Let's add DI to the Agent

Reply:
Just to clarify @donaldheppner - do you mean adding `MetricRepository` / `BatchInspector` as a dependency for the action `ColumnDescriptiveMetricsAction`? I think this would allow for cleaner tests. Or do you mean deeper changes to the Agent?

Reply:
yeah, that's right. and we can manually init in the agent (with the DI as a follow on later)

Reply:
Done!